taskfiles/rp.yml

```diff
@@ -107,10 +107,10 @@ tasks:
           '{{.SRC_DIR}}'
       fi
     - rm '{{.SRC_DIR}}/.dockerignore'
-    - cmd: "if [ '{{.CI}}' == 'true' ] && [ '{{.BRANCH_NAME}}' = 'dev' ]; then docker login --username {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}; fi"
+    - cmd: "if [ '{{.CI}}' == 'true' ] && [ '{{.BRANCH_NAME}}' = 'dev' ] && [ '{{.BUILD_TYPE}}' = 'release' ] && [ '{{.COMPILER}}' = 'clang' ]; then docker login --username {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}; fi"
```

Comment:
i don't know why this line needs to change. logging in has no effect really.

Reply:
it needs to be done so that the next step can push the image. it is on a separate line so that it can be silenced